tests/bluetooth/bluetooth/src/bluetooth.c

```diff
@@ -41,22 +42,18 @@ static void driver_init(void)
 	bt_hci_driver_register(&drv);
 }
 
-void main(void)
+void test_bluetooth_entry(void)
 {
-	int ret, ret_code;
-
-	TC_START("bluetooth");
 	driver_init();
 
-	ret = bt_enable(NULL);
-	if (ret == EXPECTED_ERROR) {
-		ret_code = TC_PASS;
-	} else {
-		ret_code = TC_FAIL;
-	}
-
-	TC_END(ret_code, "%s - %s.\n", ret_code == TC_PASS ? PASS : FAIL,
-		   __func__);
+	zassert_true(bt_enable(NULL),
```

Comment:
shouldn't this be zassert_true(bt_enable(NULL) == EXPECTED_ERROR

Reply:
Opps... Thanks catching it.